Source/WebCore/css/CSSValueList.h

```diff
@@ -29,8 +29,8 @@ namespace WebCore {
 
 class CSSValueList : public CSSValue {
 public:
-    typedef Vector<Ref<CSSValue>, 4>::iterator iterator;
-    typedef Vector<Ref<CSSValue>, 4>::const_iterator const_iterator;
+    using const_iterator = Vector<Ref<CSSValue>, 4>::const_iterator;
+    using iterator = const_iterator;
```

Comment:
Do you need `using iterator = const_iterator;` if nothing is using it?

Reply:
Not sure. Not really sure why we have the type `const_iterator` either. I think it’s a good pattern to have a type `iterator` but not sure why exactly.

Reply:
I think maybe we should just use `auto` for the return type for `begin` and `end` and get rid of both these type names.

Reply:
One reason to have the `iterator` / `const_iterator` typedefs/using declarations is that it can help / is sometimes required for use in one of the c++ standard libraries generic algorithms, and in the future (that is, when we can upgrade to compiler versions that support it), this is something that be checked for using c++ concepts - see https://en.cppreference.com/w/cpp/named_req/Container for the rather big list of things. 

I also happen to think having them is a bit of useful documentation to readers.